pandas/tests/indexes/common.py

```diff
@@ -539,10 +539,10 @@ def test_delete_base(self, indices):
         assert result.name == expected.name
 
         length = len(indices)
-        msg = r"delete\(\) missing 1 required positional argument: 'loc'"
-        with pytest.raises(TypeError, match=msg):
+        msg = f"index {length} is out of bounds for axis 0 with size {length}"
+        with pytest.raises(IndexError, match=msg):
             # either depending on numpy version
-            indices.delete()
+            indices.delete(length)
```

Comment:
are we testing the missing-argument case somewhere else?

i think the "either depending..." comment is obsolete